web/src/app/workplace/components/task-attribute-value/task-attribute-value.component.scss

```diff
@@ -5,7 +5,7 @@
   &__button--primary {
     background-color: $aquamarine;
     color: white;
-    margin-top: 16px;
+    margin: 16px 0;
```

Comment:
this means margin top and bottom 16px, margin left and right 0. It's different from just margin top, is just intended?

Reply:
When I wrote the code, it was intended but it seems not to be needed anymore. I remove it.